lncfg/protocol_experimental_on.go

```diff
@@ -3,7 +3,23 @@
 
 package lncfg
 
+import "github.com/lightningnetwork/lnd/lnwire"
+
 // ExperimentalProtocol is a sub-config that houses any experimental protocol
 // features that also require a build-tag to activate.
+//
+//nolint:lll
 type ExperimentalProtocol struct {
+	SetFeature []uint64 `long:"set-feature" description:"a protocol feature bit to set in our node's announcement"`
+}
+
+// SetFeatureBit returns a set of protocol feature bits to add to the node's
+// announcement.
+func (p ExperimentalProtocol) SetFeatureBit() []lnwire.FeatureBit {
```

Comment:
The method is named `SetFeatureBit`, but isn't this more of a get function? Perhaps the word 'custom' can be used around here as well.